spec/requests/care_plan_spec.rb

```diff
@@ -0,0 +1,46 @@
+RSpec.describe "Care Plan", type: :request do
+  describe '#index' do
+    context 'when user is signed in' do
+      let(:user) { create(:user) }
+      before do
+        sign_in user
+        get '/care_plan'
+      end
+
+      it 'renders template' do
+        expect(response).to render_template(:index)
```

Comment:
As @bennpham has pointed out

> `render_template` is only available with `rails-controller-test` gem so let's just change this to `be_successful`

So let's change all `render_template` expectations

Reply:
@tyler-wel  Thanks for the comments, I can't assign myself

Reply:
It's changed